pkg/synthesis/model/symbolicexpr/groupBasedInternalResource.go

```diff
@@ -6,6 +6,7 @@ import (
 	"github.com/np-guard/models/pkg/netset"
 	"github.com/np-guard/vmware-analyzer/pkg/collector"
 	"github.com/np-guard/vmware-analyzer/pkg/configuration"
+	resources "github.com/np-guard/vmware-analyzer/pkg/configuration/generated"
```

Comment:
```suggestion
	nsx "github.com/np-guard/vmware-analyzer/pkg/configuration/generated"
```

be consistent with import name in other places.

Reply:
almost anywhere used as `nsx`:

```
 git grep "configuration\/generated"
pkg/analyzer/explanation_test.go:       nsx "github.com/np-guard/vmware-analyzer/pkg/configuration/generated"
pkg/collector/collectApi.go:    nsx "github.com/np-guard/vmware-analyzer/pkg/configuration/generated"
pkg/collector/collector.go:     nsx "github.com/np-guard/vmware-analyzer/pkg/configuration/generated"
pkg/collector/data_model.go:    nsx "github.com/np-guard/vmware-analyzer/pkg/configuration/generated"
pkg/collector/resources_container_model.go:     nsx "github.com/np-guard/vmware-analyzer/pkg/configuration/generated"
pkg/collector/trace_flow.go:    nsx "github.com/np-guard/vmware-analyzer/pkg/configuration/generated"
pkg/collector/trace_flow_data_model.go: nsx "github.com/np-guard/vmware-analyzer/pkg/configuration/generated"
pkg/configuration/config.go:    nsx "github.com/np-guard/vmware-analyzer/pkg/configuration/generated"
pkg/configuration/dfw/category.go:      nsx "github.com/np-guard/vmware-analyzer/pkg/configuration/generated"
pkg/configuration/dfw/dfw.go:   nsx "github.com/np-guard/vmware-analyzer/pkg/configuration/generated"
pkg/configuration/dfw/rule.go:  nsx "github.com/np-guard/vmware-analyzer/pkg/configuration/generated"
pkg/configuration/dfw/rule_str.go:      nsx "github.com/np-guard/vmware-analyzer/pkg/configuration/generated"
pkg/configuration/filter_resources.go:  nsx "github.com/np-guard/vmware-analyzer/pkg/configuration/generated"
pkg/configuration/parser.go:    nsx "github.com/np-guard/vmware-analyzer/pkg/configuration/generated"
pkg/configuration/topology.go:  nsx "github.com/np-guard/vmware-analyzer/pkg/configuration/generated"
pkg/configuration/topology/segment.go:  nsx "github.com/np-guard/vmware-analyzer/pkg/configuration/generated"
pkg/data/examples.go:   nsx "github.com/np-guard/vmware-analyzer/pkg/configuration/generated"
pkg/data/examples_generator.go: nsx "github.com/np-guard/vmware-analyzer/pkg/configuration/generated"
pkg/data/group_generator.go:    nsx "github.com/np-guard/vmware-analyzer/pkg/configuration/generated"
pkg/data/groups_examples.go:import nsx "github.com/np-guard/vmware-analyzer/pkg/configuration/generated"
pkg/synthesis/model/symbolicexpr/model.go:      resources "github.com/np-guard/vmware-analyzer/pkg/configuration/generated"
pkg/synthesis/model/symbolicexpr/symbolicexpr_GroupTag_test.go: nsx "github.com/np-guard/vmware-analyzer/pkg/configuration/generated"
pkg/synthesis/model/symbolicexpr/tagTerm.go:    resources "github.com/np-guard/vmware-analyzer/pkg/configuration/generated"
```

Reply:
should be changed in `symbolicexpr/model.go` and in `symbolicexpr/tagTerm.go`